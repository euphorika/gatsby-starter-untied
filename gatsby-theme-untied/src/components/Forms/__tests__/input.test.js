import React from "react"
import renderer from "react-test-renderer"
import Input from "../input"

describe("Input", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Input name="Name" />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it("renders required input correctly", () => {
    const tree = renderer
      .create(<Input name="Name" options={{ required: true }} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
