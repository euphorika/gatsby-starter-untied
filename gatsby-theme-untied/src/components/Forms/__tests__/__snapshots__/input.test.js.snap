// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input renders correctly 1`] = `
<div
  className="element-container css-0"
>
  <input
    className="element"
    name="Name"
    type="text"
  />
  <span
    className="after-form-element"
  />
</div>
`;

exports[`Input renders required input correctly 1`] = `
<div
  className="element-container css-0"
>
  <input
    className="element"
    name="Name"
    required={true}
    type="text"
  />
  <span
    className="after-form-element"
  />
</div>
`;
